packages/ai-api/README.md

```diff
@@ -1,34 +1,42 @@
 # @sap-ai-sdk/ai-api
 
-This package provides tools to manage your scenarios and workflows in SAP AI Core.
+This package provides tools to manage scenarios and workflows in SAP AI Core.
 
 - Streamline data preprocessing and model training pipelines.
 - Execute batch inference jobs.
-- Deploy inference endpoints for your trained models.
-- Register custom Docker registries, sync AI content from your own git repositories, and register your own object storage for training data and model artifacts.
+- Deploy inference endpoints for trained models.
+- Register custom Docker registries, sync AI content from your Git repositories, and register object storage for training data and model artifacts.
```

Comment:
Since it seems we are trying to remove the "you" form, perhaps consider
```
Register custom Docker registries, sync AI content from Git repositories
```
instead